tests/game/test_buildings.py

```diff
@@ -29,6 +29,8 @@
 from horizons.world.production.producer import Producer
 from tests.game import game_test, settle
 
+import pytest
```

Comment:
This import is in the wrong order.

Try to make it a habit of running pycodestyle and isort against your changes. It should be between itertools (stdlib) and horizons (our code) imports.